### PR TITLE
added rule 5 and 6

### PR DESCRIPTION
**Pull request by botuser-jfit-rules**

### CODE REVIEW

**Comment by ayushgithub** [comment]
Can one of the admins verify this patch?

**Comment by ayushgithub** [comment]
ok to test

File: rules/rule6.py [hunk 1]
```diff
@@ -0,0 +1 @@
+assert (1 == 3), 'this is a test that passed'
```
Comment (ayushgithub) [inline rules/rule6.py]: Traceback (most recent call last):
  File "rules/rule6.py", line 1, in <module>
    assert (1 == 3), 'this is a test that passed'
AssertionError: this is a test that passed

File: rules/rule5.py [hunk 2]
```diff
@@ -0,0 +1 @@
+assert (1 == 2), 'this is a test that passed'
```
Comment (ayushgithub) [inline rules/rule5.py]: Traceback (most recent call last):
  File "rules/rule5.py", line 1, in <module>
    assert (1 == 2), 'this is a test that passed'
AssertionError: this is a test that passed

File: rules/rule6.py (hunk 1 above)
Comment (ayushgithub) [inline rules/rule6.py]: Traceback (most recent call last):
  File "rules/rule6.py", line 1, in <module>
    assert (1 == 3), 'this is a test that passed'
AssertionError: this is a test that passed

Comment (eeishaan) [inline rules/rule6.py]: Test

**Pull request merged**